generators/languages/index.js

```diff
@@ -13,9 +13,39 @@ var LanguagesGenerator = generators.Base.extend({});
 
 util.inherits(LanguagesGenerator, scriptBase);
 
+var configOptions = {};
+
 module.exports = LanguagesGenerator.extend({
     constructor: function() {
         generators.Base.apply(this, arguments);
+
+        configOptions = this.options.configOptions || {};
+
+        // This makes it possible to pass `languages` by argument
+        this.argument('languagesArgument', {
+            type: Array,
+            required: false,
+            description: 'Languages'
+        });
+
+        // This adds support for a `--skip-wiredep` flag
+        this.option('skip-wiredep', {
+            desc: 'Skip the wiredep step',
+            type: Boolean,
+            defaults: false
+        });
+
+        this.skipWiredep = this.options['skip-wiredep'];
+
+
+        // Validate languages passed as argument
+        for (var id in this.languagesArgument) {
```

Comment:
It works as it is since I guess languages argument is allways an empty array when not passing arguments. Do you want me to change it anyway?